src/programs/ps-matrix/ps-matrix

```diff
@@ -139,27 +145,39 @@ def usage():
     print('       print data values as a preflop holdem matrix')
     print('       ')
     print('       options:')
-    print('       --hand-column,h <N>          column number to find hand, default=0')
-    print('       --value-column,v <N>         column number to find value, default=1')
+    print(
```

Comment:
We should not wrap here, makes it harder to read.

Reply:
Yes, there is, but the real problem is the usage of getopt.
If you merge this,
I will refactor everything with argparse and make it python 3 compatible
(also fix the mix of print statements and functions)